src/administration/snapshot-and-restore.md

```diff
@@ -48,13 +48,27 @@ $ platform snapshot:restore 92c9a4b2aa75422efb3d
 > **note**
 > You need "admin" role to restore your environment from a snapshot.
 
-## Automated snapshots
+## Snapshots and downtime
 
-No snapshot is triggered automatically on Platform.sh Standard. You can trigger your snapshot via the Web Interface or via the CLI.
+A snapshot does cause a momentary pause in service. We recommend running during non-peak hours for your site.
 
-Snapshots may be triggered from an automated system, such as cron, Jenkins, or another CI service by calling the CLI.  To enable backups automatically using just Platform.sh, see the section on [API tokens](/gettingstarted/cli/api-tokens.md) for installing the CLI in the app container.
+## Automating snapshots
 
-Once the CLI is installed and an API token configured you can add a cron task to run once a day and trigger a snapshot.  The CLI will read the existing environment variables in the container and default to the project and environment it is running on.  However, in most cases such backups are only useful on the `master` production environment.  That can be achieved like so:
+Snapshots are not triggered automatically on Platform.sh Professional. 
+
+Snapshots may be triggered from an automated system by calling the CLI, such Jenkins, or another CI service, or by installing the CLI tool into your application container and triggering the snapshot via cron.
```

Comment:
This doesn't quite flow, and is definitely missing an `as` before "Jenkins".  How about

> Snapshots may be triggered by calling the CLI from an automated system such as Jenkins or another CI service, or by installing the CLI tool into your application container and triggering the snapshot via cron.